objects/src/testing/block.rs

```diff
@@ -35,34 +35,62 @@ impl BlockHeader {
                 .collect::<Vec<_>>(),
         )
         .expect("failed to create account db");
-        let acct_root = acct_db.root();
+        let account_root = acct_db.root();
 
         #[cfg(not(target_family = "wasm"))]
-        let (prev_hash, chain_root, nullifier_root, note_root, tx_hash, proof_hash, timestamp) = {
+        let (
+            prev_hash,
+            chain_root,
+            nullifier_root,
+            note_root,
+            tx_hash,
+            kernel_root,
+            proof_hash,
+            timestamp,
+        ) = {
             let prev_hash = rand_array::<Felt, 4>().into();
             let chain_root = chain_root.unwrap_or(rand_array::<Felt, 4>().into());
             let nullifier_root = rand_array::<Felt, 4>().into();
             let note_root = note_root.unwrap_or(rand_array::<Felt, 4>().into());
             let tx_hash = rand_array::<Felt, 4>().into();
+            let kernel_root = rand_array::<Felt, 4>().into();
```

Comment:
Wouldn't this mean that if we try to run a transaction against this block header we won't be able to execute any kernel procedures? Does this not create problems?

Reply:
This mock `BlockHeader` is used only in the serde test, so we just need to make sure that it was serialized and deserialized properly. As far as I can see we don't use this mock block header creator anywhere else, so it won't create any problems.